vcd/resource_vcd_vapp_network.go

```diff
@@ -236,7 +236,12 @@ func resourceVappNetworkRead(d *schema.ResourceData, meta interface{}) error {
 			d.Set("dns2", c.IPScopes.IPScope[0].DNS2)
 			d.Set("dnsSuffix", c.IPScopes.IPScope[0].DNSSuffix)
 		}
-		d.Set("guest_vlan_allowed", &c.GuestVlanAllowed)
+		// TODO adjust when we have option to switch between API versions or upgrade the default version
```

Comment:
Is this TODO still actual? I'm reading the comment below and the code and it looks like it works on all versions, no?

Reply:
It does not. This was the problem I discovered. Currently we are using v27 and the response is not being returned.

In this case we're not explicitly setting it if the response was not returned - we just rely that it was set as in the configuration. I did check in UI and enabling it actually works, but retrieving with v27 doesn't.

Reply:
In other words, we can't retrieve it from the newer vCDs only because we're handshaking with the older v27?

Reply:
Yes. I have tested this on vCD 9.5 and it was the case. If you use API v31.0 - this field is returned (and that's the version HTML5 UI is using). If you use v27 - this field is not returned.

Reply:
Got it, thanks!